csrankings-p.csv

```diff
@@ -497,6 +498,7 @@ Peixiang Zhao,Florida State University,http://www.cs.fsu.edu/~zhao,ajRhtAkAAAAJ
 Peiyi Tang,University of Arkansas - Little Rock,http://www.ualr.edu/pxtang,NOSCHOLARPAGE
 Peizhao Hu,Rochester Institute of Technology,https://www.cs.rit.edu/~ph,w4PoXGMAAAAJ
 Pejman Lotfi-Kamran,IPM,http://cs.ipm.ac.ir/~plotfi,UZIKgasAAAAJ
+Pekka Jääskeläinen,Tampere University,https://www.tuni.fi/en/pekka-jaaskelainen,wQHcOU0AAAAJ

```

Comment:
Dead home page.

Reply:
Thank you! We will replace it with

https://researchportal.tuni.fi/en/persons/pekka-j%C3%A4%C3%A4skel%C3%A4inen